Packs/FeedOpenCTI/Integrations/FeedOpenCTI/FeedOpenCTI.py

```diff
@@ -156,7 +156,7 @@ def main():
         max_fetch = 500
 
     try:
-        client = OpenCTIApiClient(base_url, api_key, ssl_verify=params.get('insecure'))
+        client = OpenCTIApiClient(base_url, api_key, ssl_verify=params.get('insecure'), log_level='error')
```

Comment:
Why have we changed this?

Reply:
because default log_level is : info. If we left it as info, it prints every OpenCTIApiClient info log to stderr. This returns error to war room, after each command.